src/content/advanced/security-policy-enforcement/index.md

```diff
@@ -0,0 +1,1073 @@
+---
+linkTitle: Security policy enforcement
+title: Security policy enforcement
+description: This article describes the security policies enforced in a cluster and how to resolve failing resources.
+weight: 60
+menu:
+  main:
+    parent: advanced
+user_questions:
+ -  Why won't my workload deploy?
+ -  How do I fix a Kyverno policy violation?
+ -  How can I exclude a workload from a Kyverno policy?
+ -  What security policies are enforced in my cluster?
+ -  What are Pod Security Standards (PSS)?
+ -  How do I migrate from PSPs to PSS?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-shield
+last_review_date: 2023-03-21
+---
+
+<!-- {{< platform_support_table aws="alpha=v17.2.0" aws="ga=v17.4.0">}} -->
+
+*Note: this content is applicable only to clusters running Kyverno, with the PSS `baseline` and `restricted` policies deployed in `enforce` mode. This will become the default in future Giant Swarm clusters, but can already be enabled on demand. This page will be updated to indicate the platform version where this content becomes the standard.*
```

Comment:
```suggestion
__Note__: this content is applicable only to clusters running Kyverno, with the PSS `baseline` and `restricted` policies deployed in `enforce` mode. This will become the default in future Giant Swarm clusters, but can already be enabled on demand. This page will be updated to indicate the platform version where this content becomes the standard.
```